src/lando/conftest.py

```diff
@@ -928,16 +930,26 @@ def _create_hg_commit(clone_path: Path) -> str:
 
 
 @pytest.fixture
-def conduit_permissions():
+def to_permissions() -> Callable:
```

Comment:
Maybe there should be a mention of profile permissions, either in the name or docstring.